libs/src/api/serviceProvider.js

```diff
@@ -79,7 +81,7 @@ class ServiceProvider extends Base {
       blacklist
     })
 
-    const { primary, secondaries, services } = await creatorNodeSelection.select()
+    const { primary, secondaries, services } = await creatorNodeSelection.select(performSyncCheck)
```

Comment:
why perform a sync check at replica set selection time...?

Reply:
there is no sync check performed! i set the flag to false for users on signup: https://github.com/AudiusProject/audius-protocol/pull/1142/files#diff-a34533690240ad151f75e407a02a875ee287c63c60bbdf814955e6ecbce52b13R237-R241